server/controllers/api/accounts.ts

```diff
@@ -72,6 +74,15 @@ accountsRouter.get('/:accountName/video-channels',
   asyncMiddleware(listAccountChannels)
 )
 
+accountsRouter.get('/:accountName/video-channel-syncs',
+  asyncMiddleware(accountNameWithHostGetValidator),
+  paginationValidator,
+  videoChannelSyncsSortValidator,
+  setDefaultSort,
+  setDefaultPagination,
```

Comment:
@Chocobozzz I assumed that channel sync information is not confidential and can be exposed publicly. But I am not 100% sure of this choice.

Do you think the access should be restricted? Or is this fine?

Reply:
I think you can restrict it, just in case there are credentials in the URL (we can import other things than youtube channels)

Reply:
That point is interesting, although the user should not give URL with credentials because it can be seen by root.

But let's rather restrict and raise good arguments if we want to allow anyone to see the synchronizations.